### PR TITLE
Interop - Onboard EAP XP

### PR DESCRIPTION
**Pull request by oharan2**
Depends on #44902

### CODE REVIEW

**Comment by oharan2** [comment]
/pj-rehearse pull-ci-jboss-eap-qe-openshift-eap-tests-7.4.x-eap-ocp4.15-lp-interop-images

**Comment by oharan2** [comment]
/pj-rehearse pull-ci-jboss-eap-qe-openshift-eap-tests-7.4.x-eap-ocp4.15-lp-interop-images

**Comment by oharan2** [comment]
/pj-rehearse pull-ci-jboss-eap-qe-openshift-eap-tests-pit-7.4.x-eap-ocp4.15-lp-interop-images

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-jboss-eap-qe-openshift-eap-tests-pit-7.4.x-eap-ocp4.15-lp-interop-eap-74-xp-interop-aws

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-jboss-eap-qe-openshift-eap-tests-pit-7.4.x-eap-ocp4.15-lp-interop-eap-74-xp-interop-aws

File: ci-operator/jobs/jboss-eap-qe/openshift-eap-tests/jboss-eap-qe-openshift-eap-tests-pit-7.4.x-periodics.yaml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    org: jboss-eap-qe
+    repo: openshift-eap-tests
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws-cspi-qe
+    ci-operator.openshift.io/variant: eap-ocp4.15-lp-interop
+    ci.openshift.io/generator: prowgen
+    job-release: "4.15"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-jboss-eap-qe-openshift-eap-tests-pit-7.4.x-eap-ocp4.15-lp-interop-eap-74-xp-interop-aws
```
Comment (calebevans) [inline ci-operator/jobs/jboss-eap-qe/openshift-eap-tests/jboss-eap-qe-openshift-eap-tests-pit-7.4.x-periodics.yaml:19]: I think you may be missing a Slack config here, unless you don't need Slack pass/fail notifications on this job.

**Review by calebevans — COMMENTED** [review]
Just a small thing I noticed, if you don't need it, then it looks good to me. Just let me know if you want me to approve without the Slack configuration.

File: ci-operator/jobs/jboss-eap-qe/openshift-eap-tests/jboss-eap-qe-openshift-eap-tests-pit-7.4.x-periodics.yaml (hunk 1 above)
Comment (Sau1506mya) [reply ci-operator/jobs/jboss-eap-qe/openshift-eap-tests/jboss-eap-qe-openshift-eap-tests-pit-7.4.x-periodics.yaml:19]: Yeah @calebevans , we are left with the addition of `reporter_config` to add the slack channel configuration. 
@oharan2 , once you are back please add
`reporter_config:
    slack:
      channel: '#eap-ocp-ci-results'
      job_states_to_report:
      - success
      - failure
      - error
      report_template: '{{if eq .Status.State "success"}} :success: Job *{{.Spec.Job}}*
        ended with *{{.Status.State}}*. <{{.Status.URL}}|View logs> {{else}} :dumpster-fire:
        Job *{{.Spec.Job}}* ended with *{{.Status.State}}*. <{{.Status.URL}}|View
        logs> {{end}}'`
I already confirmed it from PQE.

**Comment by oharan2** [comment]
/pj-rehearse pull-ci-jboss-eap-qe-openshift-eap-tests-pit-7.4.x-eap-ocp4.15-lp-interop-images

**Comment by oharan2** [comment]
/pj-rehearse periodic-ci-jboss-eap-qe-openshift-eap-tests-pit-7.4.x-eap-ocp4.15-lp-interop-eap-74-xp-interop-aws

**Review by calebevans — APPROVED** [review]
/lgtm

**Pull request merged**